test/test_geocode.py

```diff
@@ -122,6 +122,8 @@ def test_collapse_hierarchy():
     """Collapse spatial objects that are members of each others' hierarchies 
     into a single object.
     """
+    # FIXME: See FIXME comment in geocode.merge_related().
```

Comment:
@markbreedlove should this be removed too if it is referring to the comment  you noted above?